flow/layers/performance_overlay_layer.cc

```diff
@@ -134,16 +137,22 @@ void PerformanceOverlayLayer::Paint(PaintContext& context) const {
   SkScalar width = paint_bounds().width() - (padding * 2);
   SkScalar height = paint_bounds().height() / 2;
   auto mutator = context.state_stack.save();
+  // Cached storage for vertex output.
+  std::vector<DlPoint> vertices_storage;
+  std::vector<DlColor> color_storage;
```

Comment:
Weren't these supposed to be cached somewhere? It looks like they are recreated on every frame?

Reply:
Or is this just re-use between the 2 graphs?

Reply:
Just between the two graphs. originally I was much more ambitious but it turns out that we rebuild the layer each frame so there isn't a good place to put the storage.